src/Nethermind/Nethermind.Consensus/Processing/BlockExtensions.cs

```diff
@@ -1,14 +1,17 @@
 // SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
+using System;
 using System.Collections.Generic;
+using System.Text;
+using Nethermind.Config;
 using Nethermind.Consensus.Producers;
 using Nethermind.Core;
 using Nethermind.State.Proofs;
 
 namespace Nethermind.Consensus.Processing
 {
-    internal static class BlockExtensions
+    public static class BlockExtensions
```

Comment:
Changed this to `public` so that it can be accessed by unit test. Is there a better way to do this? I tried [changing the .csproj](https://stackoverflow.com/a/73261024) without success

Reply:
Solution to this is adding: `[assembly: InternalsVisibleTo("Nethermind.Consensus.Test")]`